apps/hash-ai-worker-ts/src/activities/infer-entities/propose-entities.ts

```diff
@@ -371,7 +371,10 @@ export const proposeEntities = async (params: {
                       try {
                         await graphApiClient.validateEntity(validationActorId, {
                           entityTypes: [entityTypeId],
-                          profile: "draft",
+                          components: {
+                            numItems: false,
```

Comment:
This fixes the issue where links always fail validation at this stage (since we don't have the right/left entities saved to the db)
```suggestion
                            linkData: false,
                            numItems: false,
```

Reply:
Great, thank you!